raiden/messages.py

```diff
@@ -730,8 +750,7 @@ class RevealSecret(SignedMessage):
     cmdid = messages.REVEALSECRET
 
     def __init__(self, message_identifier: MessageID, secret: Secret):
-        super().__init__()
-        self.message_identifier = message_identifier
+        SignedRetrieableMessage.__init__(self, message_identifier)
```

Comment:
This explicit way of calling the super is used when you have multiple parent classes... in this case, we have 1 so `super` would be enough... same for any other place using this.

Reply:
It's not necessary even with multiple base classes (unless you absolutely must control the order in which the parent methods are called). The MRO algorithm used by Python makes sure that all base class methods are called (see the example in my comment above).

Reply:
please check https://circleci.com/gh/raiden-network/raiden/24764?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link -- I cannot get it to work without explicit `__init__` calls.